test/chrome-launcher-test.ts

```diff
@@ -133,6 +133,11 @@ describe('Launcher', () => {
     assert.ok(!chromeFlags.includes('--disable-extensions'));
   });
 
+  it('searches for available installations', async () => {
+    const installations = Launcher.getInstallations();
```

Comment:
assert that we found one too? :)

Reply:
I wasn't sure if that was safe given we don't currently require chrome as a dev dependency (that I can tell), so CI might not pass?

Reply:
OK I added the check --hopefully that works as intended 😆